src/helics/core/CoreBroker.hpp

```diff
@@ -232,10 +232,11 @@ class CoreBroker: public Broker, public BrokerBase {
     /** add a route to the type specific routing information and establish the connection
     @details add a route to a table, the connection information is contained in the string with the described
     identifier
-    @param rid  the identifier for the route
-    @param routeInfo  a string describing the connection info
+    @param rid the identification of the route
+    @param interfaceId an interface id code that can be used to identify the interface route should be added to in most cases this should be zero since there is only one interface
```

Comment:
```suggestion
    @param interfaceId an interface id code that can be used to identify the interface route should be added to, in most cases this should be zero since there is only one interface
```